### ISSUE TITLE
Meter only a single method

### ISSUE THREAD

**Issue by bytemonkio** [tokuhirom/spring-boot-richmetrics-filter#3]
If I want to time/meter only a single method and want to get rid of all the other info, is that possible?

**Comment by tokuhirom** [comment]
No. It's a servlet level filter. You can't do it with this method.
But you can do it with the path filter.

Such as following:

```
if (req.getPathInfo().equals("/foo")) {
}
```
